### PR TITLE
 login

### PR DESCRIPTION
**Pull request by prath-dev**
Screenshot from 2022-02-22 00-10-41

### CODE REVIEW

File: Views/login.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<body>
+    <nav>
+        <a class="active" href="..//Views/index.html">MyStore</a>
+        <input class="search-bar" type="text" placeholder="Search">
+        <a href="../Views/cart.html">Cart<span class="material-icons-outlined">
+            shopping_cart
+          </span>
+              </a>
+        <a href="..//Views/wishlist.html">WishList<span class="material-icons-outlined">
```
Comment (pnchinmay) [inline Views/login.html:22]: These won't be linked correctly

File: Views/login.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+       favorite_border
+        </span>
+          </a>
+        <a href="..//Views/login.html">Login</a>
```
Comment (pnchinmay) [inline Views/login.html:26]: same here

File: Views/login.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <input placeholder="******" class="input-box" type="password">
+            <div>
+                <input class="btn" type="button" value="Login">
+            </div>
+            <div class="checkbox">
+                <input type="checkbox">
+                <span>Remember Me</span>
+            </div>
+            <div class="sign-in">
+                <a href="..//Views/signup.html"> Already have Account? SignIn</a>
```
Comment (pnchinmay) [inline Views/login.html:45]: same here

**Review by pnchinmay — APPROVED** [review]
Everything else looks good

**Review by DevNirah7 — APPROVED** [review]
Barring changes as mentioned above by Chinmay, rest all looks good to commit!

File: Views/login.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="..//Components/Css/login.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav>
+        <a class="active" href="..//Views/index.html">MyStore</a>
+        <input class="search-bar" type="text" placeholder="Search">
+        <a href="../Views/cart.html">Cart<span class="material-icons-outlined">
```
Comment (sashaboi) [inline Views/login.html:18]: maybe putting the cart text and material icon text encapsulated in a div , and flex align and justify center might make them look more pleasant.
Onkar Deshpande

**Comment by prath-dev** [comment]
@sashaboi @DevNirah7  thx for your suggestion i will do changes in second version of my ecommerce ..

**Pull request merged**